include/abb_librws/rws_poco_client.h

```diff
@@ -345,6 +345,19 @@ class POCOClient
    */
   POCOResult webSocketRecieveFrame();
 
+  /**
+   * \brief Close the websocket connection.
```

Comment:
```suggestion
   * \brief Force close the WebSocket connection.
```
Slightly more accurate description.